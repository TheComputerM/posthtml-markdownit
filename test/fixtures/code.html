<md>
    # Heading
    
    ```js
    function () {
        console.log("Hello")
    }
    ```
</md>