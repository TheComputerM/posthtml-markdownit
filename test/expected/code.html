<h1>Heading</h1>
<pre><code class="language-js">function () {
  console.log(&quot;Hello&quot;)
}
</code></pre>
